source/Date.cpp

```diff
@@ -49,23 +52,57 @@ Date::Date(int day, int month, int year)
 // Convert a date to a string.
 const string &Date::ToString() const
 {
+	string dateFormat = Preferences::DateFormat();
```

Comment:
Use an enum class (see how the VSync preference is handled in the Preferences class.)

Reply:
Already done.